packages/contracts-bedrock/src/L1/StandardValidator.sol

```diff
@@ -87,6 +94,46 @@ contract StandardValidator {
         mipsImpl = _implementations.mipsImpl;
     }
 
+    function expectedGuardian(
+        ValidationOverrides memory _overrides,
+        string memory _errors
+    )
+        internal
+        view
+        returns (address, string memory)
+    {
+        if (_overrides.guardian != address(0)) return (_overrides.guardian, string.concat("HAS_OVERRIDES", _errors));
```

Comment:
Should change this to `OVERRIDES_GUARDIAN`, to be more specific. Same for the below.